cmd/util/flags/configflags/web_ui.go

```diff
@@ -5,8 +5,14 @@ import "github.com/bacalhau-project/bacalhau/pkg/config/types"
 var WebUIFlags = []Definition{
 	{
 		FlagName:     "web-ui",
-		ConfigPath:   types.NodeWebUI,
+		ConfigPath:   types.NodeWebUIEnabled,
 		DefaultValue: false,
 		Description:  `Whether to start the web UI alongside the bacalhau node.`,
 	},
+	{
+		FlagName:     "web-ui-port",
+		ConfigPath:   types.NodeWebUIPort,
+		DefaultValue: 80,
```

Comment:
This seems to be ignored and not set for the ConfigPath.
e.g. `listenPort, err := config.Get[int](types.NodeWebUIPort)` returns 0 instead of 80.

Reply:
This sounds like a reasonably serious problem?

Reply:
weird, let me check this out then.

Reply:
I think we forgot to run https://github.com/bacalhau-project/bacalhau/blob/main/pkg/config/types/config.go#L3

Reply:
Issue is a mixture of the above and not setting the defaults for our various configs will push a change to fix this in a sec.